public/app/features/canvas/runtime/scene.tsx

```diff
@@ -281,8 +281,7 @@ export class Scene {
   };
 
   render() {
-    const isTooltipValid = (this.tooltip?.element?.data?.links?.length ?? 0) > 0;
-    const canShowElementTooltip = !this.isEditingEnabled && isTooltipValid;
+    const canShowElementTooltip = !this.isEditingEnabled;
 
```

Comment:
Maybe something like if we go with more exclusive method for displaying tooltips

```suggestion
    const isTooltipValid = (this.tooltip?.element?.data?.links?.length ?? 0) > 0 || this.tooltip?.element?.data?.field;
    const canShowElementTooltip = !this.isEditingEnabled && isTooltipValid;
```